src/presentation/interfaces/action-trigger-event-listener.ts

```diff
@@ -0,0 +1,5 @@
+import { ActionTriggerEvent } from '../value-objects/action-trigger-event'
+
+export interface ActionTriggerEventListener {
+  listenToActionTriggerEvents(onActionTriggerEventCallback: (actionTriggerEvent: ActionTriggerEvent) => void): void
+}
```

Comment:
This interface and the implementing class seems to conceptually implement the observer pattern.
Suggestion: The interface could be suffixed with `Observer` instead of `Listener` and the method renamed to `subscribeToActionTriggerEvents`. It would also streamline the naming across frontend and backend.

Reply:
No objections here. I have renamed `RundownEventListener` and `ActionTriggerEventListener` to `RundownEventObserver` and `ActionTriggerEventObserver`